src/main/java/tutu/task/Task.java

```diff
@@ -0,0 +1,49 @@
+package tutu.task;
+
+/** Represents a task. */
+public class Task {
+    /** Description of task */
+    protected String task;
```

Comment:
I guess using name like `taskDescription` here would be more precise? Maybe it's just me, but `task` somehow gives me a feel that it is an instance of class `Task`.